beacon_node/lighthouse_network/src/rpc/protocol.rs

```diff
@@ -402,8 +429,16 @@ impl ProtocolId {
                 <MetaDataV2<T> as Encode>::ssz_fixed_len(),
             ),
             Protocol::LightClientBootstrap => RpcLimits::new(
-                <LightClientBootstrapRequest as Encode>::ssz_fixed_len(),
-                <LightClientBootstrapRequest as Encode>::ssz_fixed_len(),
+                <LightClientBootstrap<T> as Encode>::ssz_fixed_len(),
+                <LightClientBootstrap<T> as Encode>::ssz_fixed_len(),
+            ),
+            Protocol::LightClientOptimisticUpdate => RpcLimits::new(
+                <LightClientOptimisticUpdate<T> as Encode>::ssz_fixed_len(),
+                <LightClientOptimisticUpdate<T> as Encode>::ssz_fixed_len(),
+            ),
+            Protocol::LightClientFinalityUpdate => RpcLimits::new(
+                <LightClientFinalityUpdate<T> as Encode>::ssz_fixed_len(),
+                <LightClientFinalityUpdate<T> as Encode>::ssz_fixed_len(),
```

Comment:
Update rpc limits as this is no longer fixed length

Reply:
https://github.com/sigp/lighthouse/pull/3849/commits/c888923304cc5dbed7d42b9cc0a4addd821ecdef